Author: William Vera <billy@billy.com.mx>
Description: Add a new option in the man page (focused option).
--- a/scrot.1
+++ b/scrot.1
@@ -43,6 +43,9 @@
 .B -s, --select
 Interactively select a window or rectangle with the mouse.
 .TP 5
+.B -u, --focused
+Use the currently focused window.
+.TP 5
 .B -t, --thumb NUM
 generate thumbnail too. NUM is the percentage of the original size for the
 thumbnail to be.
